### DISCUSSION TITLE
RoderickShao_notebook/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80%E8%AF%BE/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BB%84%E6%88%90/Chapter%204/

### DISCUSSION THREAD

**Discussion by giscus** [RoderickShao/RoderickShao_notebook#6] [Announcements]
# RoderickShao_notebook/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80%E8%AF%BE/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BB%84%E6%88%90/Chapter%204/

RoderickShao's NoteBook

https://roderickshao.github.io/RoderickShao_notebook/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80%E8%AF%BE/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BB%84%E6%88%90/Chapter%204/

**Comment by Provence-24** [comment]
4.1 Write results to register中I型指令 若是立即数加法等，则是从ALU写到rd。ld是内存写到rd

**Reply by Provence-24** [reply]
我这条评论怎么在主页里。。
